internal/distro/fedora/distro.go

```diff
@@ -32,6 +32,10 @@ const (
 
 	//Kernel options for ami, qcow2, openstack, vhd and vmdk types
 	defaultKernelOptions = "ro no_timer_check console=ttyS0,115200n8 biosdevname=0 net.ifnames=0"
+
+	// hardcode the datastream since this is provided by the scap-security-guide rpm
+	// and is not expected to change.
+	datastream = "/usr/share/xml/scap/ssg/contentssg-fedora-ds.xml"
```

Comment:
nitpick: I think that it would be good to prefix the variable name with `oscap` or something like that. E.g. `defaultOSCAPDatastream`...

Reply:
I've added the prefix and moved this to the oscap package... imo it's slightly cleaner